warehouse/transformer/internal/types/types.go

```diff
@@ -17,7 +17,7 @@ type Metadata struct {
 	SourceCategory    string         `json:"sourceCategory"`
 	EventType         string         `json:"eventType,omitempty"`
 	RecordID          interface{}    `json:"recordId,omitempty"`
-	DestinationConfig map[string]any `json:"-"`
+	DestinationConfig map[string]any `json:"destinationConfig"`
```

Comment:
Why this change?

Reply:
We need this later for debugging. DestinationConfig contains relevant information around ```
"skipTracksTable", "skipUsersTable", "underscoreDivideNumbers", "allowUsersContextTraits",
	"storeFullEvent", "jsonPaths",```

Reply:
Do we need to ensure that we are not logging destination secrets in the compare and diff part?

Reply:
We are already making sure that the Destination Config only contain the necessary fields.
https://github.com/rudderlabs/rudder-server/blob/master/warehouse/transformer/internal/types/types.go#L33-L58